backend/functions/lib/controllers/orders/orders.controller.ts

```diff
@@ -58,14 +60,17 @@ export class OrdersController {
 
   public async getSuggestedCases(
     context: ApplicationContext,
-    caseId: string,
-  ): Promise<GetSuggestedCasesResponse> {
+  ): Promise<CamsHttpResponseInit<CaseSummary[]>> {
```

Comment:
Following the other functions, this one would be:
```suggestion
  ): Promise<GetSuggestedCasesResponse> {
```